experimental/generated/beats/fields.ecs.yml

```diff
@@ -1573,6 +1573,60 @@
         Note that when the file name has multiple extensions (example.tar.gz), only
         the last one should be captured ("gz", not "tar.gz").'
       example: png
+    - name: file.elf.creation_date
+      level: extended
+      type: date
+      description: Extracted when possible from the file's metadata. Indicates when
+        it was built or compiled. It can also be faked by malware creators.
+      default_field: false
+    - name: file.elf.exports
+      level: extended
+      type: object
+      description: List of exported element names and types
+      default_field: false
+    - name: file.elf.header
+      level: extended
+      type: object
+      description: Header information of the ELF file.
+      default_field: false
+    - name: file.elf.imports
+      level: extended
+      type: object
+      description: List of imported element names and types
+      default_field: false
+    - name: file.elf.number_program_headers
+      level: extended
+      type: long
+      description: Number of ELF Program Headers.
+      default_field: false
+    - name: file.elf.number_section_headers
```

Comment:
Shouldn't this match the number of `sections` entries? Do we really need to pull this out as a pre-aggregated value?

Reply:
Fair point. Agreed.